### PR TITLE
Fixed testing based on requests

### PR DESCRIPTION
**Pull request by clmcguire**
updated the test files to use HashRouter instead of MemoryRouter and BrowserRouter

### CODE REVIEW

**Review by DAC098 — CHANGES_REQUESTED** [review]
all of the code looks good. for some reason two files seem to be throwing errors when building.
1. `src/App.tsx` it seems to be missing the `React` import at the top which can be fixed by adding `import React from "react";`
2. `src/Home/Home.tsx` is complaining about something that it should not be. also adding `import React from "react"`;
again not sure why it is complaining but adding those seems to fix it.

**Comment by DAC098** [comment]
fix the merge conflicts with the main branch and everything should be good.

**Comment by clmcguire** [comment]
@DAC098 do you want to double check that the merge conflicts were addressed properly before I commit it? Just want to be sure

**Comment by DAC098** [comment]
yeah I can take a look

File: package.json [hunk 1]
```diff
@@ -50,5 +52,13 @@
     "rollup-plugin-postcss": "^4.0.2",
     "ts-jest": "^29.2.5",
     "typescript": "^5.7.2"
```
Comment (DAC098) [inline package.json]: there needs to be a comma after the version specification other wise the file cannot be parsed.

**Review by DAC098 — DISMISSED** [review]
the only issue I saw was with `package.json`, there is an error in the file that prevents it from being parsed properly. after that everything built, all the tests ran, and from what I noticed the app continued to function as expected.

**Review by DAC098 — APPROVED** [review]

**Pull request merged**